english/mushroom/collect_mushroom.md

```diff
@@ -6,6 +6,8 @@ Upon closer examination, you realize that they are magic mushrooms.
 You reminisce back to a recent trip to Thailand where you had a life-changing experience with these magical,
 albeit poisonous mushrooms.
 
+You wonder if these mushrooms will have the same interesting effect, whether they'll kill you, or whether they'd be tasty to cook with.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.